.npmignore

```diff
@@ -1,6 +1,9 @@
 coverage
 tests
-node_modules
```

Comment:
According to [the docs](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package), this line is not needed as the `node_modules` directory is excluded by default.